lib/DDG/Spice/CodeDay.pm

```diff
@@ -0,0 +1,56 @@
+package DDG::Spice::CodeDay;
+# ABSTRACT: Get all CodeDay events, sorted by closest to farthest
+
+use DDG::Spice;
+use Text::Trim;
+
+spice is_cached => 0;
+
+name "CodeDay";
+source "CodeDay.org";
+icon_url "https://codeday.org/assets/img/favicon.ico";
+description "Get information about a CodeDay or find the nearest one.";
+primary_example_queries "codeday near me", "when is codeday?", "codeday san diego";
+secondary_example_queries "codeday", "codeday seattle", "what is codeday bay area?";
+category "programming";
+topics "programming", "geek", "computing", "entertainment";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/CodeDay.pm";
+attribution github => ["tjhorner", "TJ Horner"],
+            twitter => "tjhorner";
+
+spice to => 'https://clear.codeday.org/api/regions/$1?term=$2&lat=$3&lng=$4&limit=50';
```

Comment:
>but it's not working properly

I noticed that on beta... it works perfectly with the DuckPAN instance on my end, weird.

>or maybe just the capitals for each state, we can easily load the file and lookup the lat long for it and make an API call.

I can add a search by ZIP on our end, I still think lat/long would be more accurate though.